dll/win32/msafd/msafd.h

```diff
@@ -133,12 +133,41 @@ typedef struct _ASYNC_DATA {
 	AFD_POLL_INFO AsyncSelectInfo;
 } ASYNC_DATA, *PASYNC_DATA;
 
-typedef struct _AFDAPCCONTEXT
+typedef struct _AFDINFOAPCCONTEXT
+{
+    LPWSAOVERLAPPED lpOverlapped;
+    LPWSAOVERLAPPED_COMPLETION_ROUTINE lpCompletionRoutine;
+    PAFD_INFO lpInfoData;
+    HANDLE hSockEvent;
+} AFDINFOAPCCONTEXT, *PAFDINFOAPCCONTEXT;
```

Comment:
Don't be shy using more readable names like: `AFD_INFO_APC_CONTEXT` or the like.